packages/dcos-integration-test/extra/test_groups.yaml

```diff
@@ -0,0 +1,79 @@
+# Purpose of this file
+# ====================
+#
+# We split integration tests into groups so that the test suite can be run in
```

Comment:
Just a quick thought.... did we think about "groups" vs. "categories"? "Group" is more neutral, but even more neutral would be "set".

It's OK to not act on this.

Reply:
I infer from "category" that there is some relationship between the tests defined.

Frankly I took the terminology from https://pypi.org/project/pytest-test-groups/ and did not think further on it.

I would be OK to act on this if you'd like.

Reply:
Okay, I think what we try to do here is introduce _arbitrary_ groups; and "category" would be misleading. So we can leave it as-is.